test/snippet/range/view/interleave.cpp

```diff
@@ -0,0 +1,26 @@
+#include <range/v3/algorithm/equal.hpp>
+
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/range/view/interleave.hpp>
+
+using namespace seqan3;
+
+int main()
+{
+//![general]
+std::string u{"FOOBARBAXBAT"};
+std::string i{"in"};
+size_t s = 3;
+std::string cmp{"FOOinBARinBAXinBATin"};
+std::string cmp_rev{"niTAB"};
+
+// pipe notation
+// explicitly call view::all
+auto v0 = u | view::interleave(s, i);
+// or, via constructor.
+auto v1 = detail::view_interleave(u, s, i);
```

Comment:
don't call detail:: here, instead `auto v1 = view::interleave(u, s, i);` should work, too!

Reply:
this was never resolved